Code/Editor/ViewportTitleDlg.cpp

```diff
@@ -257,21 +261,56 @@ void CViewportTitleDlg::SetupHelpersButton()
 
 void CViewportTitleDlg::SetupOverflowMenu()
 {
-    // Setup the overflow menu
-    QMenu* overFlowMenu = new QMenu(this);
+    // simple override of QMenu that does not respond to keyboard events
+    // note: this prevents the menu from being prematurely closed
+    class IgnoreKeyboardMenu : public QMenu
+    {
+    public:
+        IgnoreKeyboardMenu(QWidget *parent = nullptr) : QMenu(parent)
+        {
+        }
 
-    m_audioMuteAction = new QAction("Mute Audio", overFlowMenu);
-    connect(m_audioMuteAction, &QAction::triggered, this, &CViewportTitleDlg::OnBnClickedMuteAudio);
-    overFlowMenu->addAction(m_audioMuteAction);
+    private:
+        void keyPressEvent([[maybe_unused]] QKeyEvent* event) override
```

Comment:
nit: does not need the maybe_unused anymore.

Reply:
Ah yes of course good spot :+1: Will fix